packages/js-api/src/core/flow.ts

```diff
@@ -45,6 +45,10 @@ export class Flow extends Fetchable {
    * The user-facing description of the Flow, if defined at the top level of the YAML config.
    */
   public subtitle?: string
+  /**
+   * Props to pass through to the Flow Component in the React SDK.
+   */
```

Comment:
This is sort of internal right? We can consider adding `@ignore` so it wont be in the official docs?

Also folks who see this who use the SDK in non-React might be confused by this so maybe we can add a comment "Used exclusively by the @frigade/react SDK" or something.